app/Console/Commands/InstallPanel.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace PufferPanel\Console\Commands;
+
+use Illuminate\Console\Command;
+use PufferPanel\Models\User;
+
+class InstallPanel extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'panel:install';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Runs the initial panel setup: creates admin user';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $uuid = sprintf('%04x%04x-%04x-%04x-%04x-%04x%04x%04x', mt_rand(0, 0xffff), mt_rand(0, 0xffff), mt_rand(0, 0xffff), mt_rand(0, 0x0fff) | 0x4000, mt_rand(0, 0x3fff) | 0x8000, mt_rand(0, 0xffff), mt_rand(0, 0xffff), mt_rand(0, 0xffff));
```

Comment:
Probably a good idea to address the use of UUIDs like this, but copied over from the old installer just for simplicities sake.